entrypoint.d/45-geoip

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
+    set -e
```

Comment:
I see you repeat it twice... IMHO it's better to add it just once at the top of the file.

Reply:
i use one `set -e` now